### ISSUE TITLE
Instructions to install RMG 3.0 binary has issues with newer packages

### ISSUE THREAD

**Issue by goldmanm** [ReactionMechanismGenerator/RMG-Py#2047]
### Bug Description
When I follow the instructions to generate an RMG environment (using RMG binaries on ubuntu 20.04), and then I run an input file, I get the following error message:

```
Traceback (most recent call last):
  File "/home/mark/anaconda3/envs/rmg_env_repo/bin/rmg.py", line 48, in <module>
    from rmgpy.rmg.main import RMG, initialize_log, process_profile_stats, make_profile_graph
  File "/home/mark/anaconda3/envs/rmg_env_repo/lib/python3.7/site-packages/rmgpy/rmg/main.py", line 68, in <module>
    from rmgpy.rmg.listener import SimulationProfileWriter, SimulationProfilePlotter
  File "/home/mark/anaconda3/envs/rmg_env_repo/lib/python3.7/site-packages/rmgpy/rmg/listener.py", line 34, in <module>
    from rmgpy.tools.plot import SimulationPlot
  File "/home/mark/anaconda3/envs/rmg_env_repo/lib/python3.7/site-packages/rmgpy/tools/plot.py", line 37, in <module>
    mpl.use('Agg', warn=False, force=False)
TypeError: use() got an unexpected keyword argument 'warn'
```

The error seems to come form using matplotlib 3.3, which doesn't have the `warn` keyword

### How To Reproduce
Install RMG package with `conda create -c defaults -c rmg -c rdkit -c cantera -c pytorch -c conda-forge -n rmg_env rmg rmgdatabase`, make sure other rmgpy versions are not on path. Get an input file, activate the env, and type `rmg.py input.py`

### Expected Behavior

Error message shown above

### Installation Information
Describe your installation method and system information.
 - Ubuntu 20.04.1
 - binary
 - 3.0
   
   - RMG-Py: 
   - RMG-database: 

### Additional Context
I got the environment working by specifying a lower matplotlib version (shown below). A bug fix would involve updating the documentation to specify the matplotlib version:

`conda create -c defaults -c rmg -c rdkit -c cantera -c pytorch -c conda-forge -n rmg_env_3 rmg rmgdatabase "matplotlib<3.3"`

**Comment by vicknguyen1992** [comment]
Same problem, I think it's major issue, when I install RMG following by [basic installation guide](http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/index.html) this error always appear:

I have to downgrade matplotlib version to 3.2.x `
conda install -c anaconda matplotlib==3.2.1
`

**Comment by mjohnson541** [comment]
This was fixed for source in #2012 and should no longer be an issue with the v3.1.0 binaries just released.
